### ISSUE TITLE
wagl container is 685.17 MB

### ISSUE THREAD

**Issue by funkytaco** [ahmetalpbalkan/wagl#9]
My wagl container is 685.17 MB.

Does it really need to be this big?

It looks like the latest one is 719 MB

https://imagelayers.io/?images=ahmet%2Fwagl:latest

**Comment by ahmetb** [comment]
@funkytaco not really, it actually can be 15 mb since it's written in go. However we need to manually pack and publish that (or perhaps download from github releases section, but we don't have a release yet). I don't want the image to be shown as non-automated build repository on Docker Hub, so I just based off on golang image.

Perhaps we can do a Dockerfile.dev for build/test and then Dockerfile that `wget`s the binary from github releases section so it can be way smaller.
